Makefile.am

```diff
@@ -3,6 +3,60 @@ SUBDIRS = po
 # Our Autoconf macros are kept in the m4/ directory
 ACLOCAL_AMFLAGS = -I m4
 
+# Helper C library
+girdir = $(datadir)/gir-1.0
+typelibdir = $(libdir)/girepository-1.0
+
+lib_LTLIBRARIES =
+gir_DATA = 
+typelib_DATA = 
+
+CLEANFILES = 
+
+lib_LTLIBRARIES += \
+	libeos-discovery-feed-1.la \
+	$(NULL)
+
+libeos_discovery_feed_1_la_SOURCES = \
+	libfeed/feed-text-sanitization.c \
+	libfeed/feed-text-sanitization.h \
+	$(NULL)
+
+libeos_discovery_feed_1_la_CPPFLAGS = \
+	$(EOS_DISCOVERY_FEED_CFLAGS) \
+	-I$(abs_top_builddir)/libfeed \
+	$(NULL)
+
+libeos_discovery_feed_1_la_LIBADD = \
+	$(EOS_DISCOVERY_FEED_LIBS) \
+	$(NULL)
+
+# GObject-Introspection
+include $(INTROSPECTION_MAKEFILE)
+INTROSPECTION_GIRS = $(NULL)
+INTROSPECTION_SCANNER_ARGS = --add-include-path=$(srcdir) --warn-all
+INTROSPECTION_COMPILER_ARGS = --includedir=$(srcdir)
+
+if HAVE_INTROSPECTION
+introspection_sources = \
+	$(libeos_discovery_feed_1_la_SOURCES) \
+	$(NULL)
+
+EosDiscoveryFeed-1.0.gir: libeos-discovery-feed-1.la
+EosDiscoveryFeed_1_0_gir_INCLUDES = EosShard-0 GObject-2.0 Gio-2.0 GLib-2.0
```

Comment:
We don't use these types yet, but probably will soon.

Reply:
Then let's not add them yet, but soon :-)